lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -855,6 +856,81 @@ static void optimizeBatchNorm(Function *F) {
   } // For all nodes in the graph.
 }
 
+/// Check if the input of this method (vector of NodeValues) can be
+/// legal input of CreateConcat function.
+/// \returns the dimension where the Concat will take place. If not
+/// concatenatable, return -1.
+static int getConcatDimensionForInputs(std::vector<NodeValue> inputs,
```

Comment:
do not pass vector by value, use ArrayRef or const vector&

Reply:
done